system-uptime-tracking/xml/art_system-uptime-tracking.xml

```diff
@@ -0,0 +1,375 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+  <!ENTITY productnamex "<phrase xmlns='http://docbook.org/ns/docbook' os='sles;sled;osuse'>&productname;</phrase><phrase xmlns='http://docbook.org/ns/docbook' os='article'>&sls;</phrase>">
+   <!ENTITY productnumberx "<phrase xmlns='http://docbook.org/ns/docbook' os='sles;sled;osuse'>&productnumber;</phrase>">
+]>
+<article xml:id="article-system-uptime-tracking"
+         xmlns="http://docbook.org/ns/docbook"
+         version="5.0"
+         xml:lang="en"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>System Uptime Tracking</title>
+  <titleabbrev>System uptime tracking in &scc; and &rmt;</titleabbrev>
+  <info><productname>&productnamex;</productname>
+    <productnumber>&productnumberx;</productnumber><date>
+<?dbtimestamp format="B d, Y" ?></date>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker>
+        <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
+        <dm:product>Documentation</dm:product>
+        <dm:component>SLE Family</dm:component>
+        <dm:assignee>tbazant@suse.com</dm:assignee>
+      </dm:bugtracker>
+      <dm:editurl>https://github.com/SUSE/doc-unversioned/edit/main/system-uptime-tracking/xml/</dm:editurl>
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+    <abstract>
+      <para>
+        The system uptime tracking feature monitors when a client system is up.
+        It works on an hourly basis&mdash;the system is considered up if it was
+        running at any part of the given hour. The information is stored in
+        &scc; (SCC) and Repository Mirroring Tool (RMT), and serves as a source
+        for analytic purposes.
+      </para>
+    </abstract>
+  </info>
+  <sect1 xml:id="uptime-requirements">
+    <title>Requirements</title>
+
+    <itemizedlist>
+      <listitem>
+        <para>
+          The version of the <package>suseconnect-ng</package> package has to be
+          1.10 or newer. Verify it by running <command>suseconnect
+          --version</command>.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          If the client system is behind RMT, the version of the
+          <package>rmt-server</package> package has to be 2.17 or newer. Verify
```

Comment:
Maybe it's a bit ambiguous that the rmt-server package has to be 2.17 or newer _on the RMT server itself_, and not on the client system. (The client system does not need  to have the rmt-server package installed at all)